bucket/libgen-desktop.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": "1.4.1",
+    "description": "Desktop for Library Gensis",
+    "homepage": "https://github.com/libgenapps/LibgenDesktop",
+    "license": "Unknown",
+    "notes": [
+        "Require Windows Vista SP2 or later and .NET Framework 4.5 (Included in Windows 8 or later by default) or higher to run.",
+        "See also https://wiki.mhut.org/software:libgen_desktop for details."
+    ],
+    "pre_install": [
+        "Out-File -FilePath \"$dir\\libgen.config\" -Encoding ascii",
+        "Out-File -FilePath \"$dir\\downloads.json\" -Encoding ascii"
+    ],
```

Comment:
The proper way to do this is like this:
https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/xmplay.json#L8-L12